samples/grid-pro/demo/cell-editing/demo.css

```
@@ -4,8 +4,8 @@
     padding: 8px;
 }
 
-.highcharts-description {
-    padding: 0 20px;
+#changelog {
```

```suggestion
#changelog:not(:empty) {
    background: #fbfbfb;
    border-radius: 5px;
    padding: 10px 15px;
    border: 1px solid #858585;
```
Just making it a little more styled and prominent